drone_simulation/drone_simulation/trajectoryGen.py

```diff
@@ -69,6 +69,72 @@ def trajectoryGenie(pos0, vel0, acc0, posf, velf, accf, Tf, numPlotPoints):
     return traj, position_x, position_y, position_z
 
 
+def trajectoryGenieAccel(pos0, vel0, acc0, posf, velf, accf, Tf, numPlotPoints):
+    """
```

Comment:
Do you think it would be better to use the traj object returned by the TrajectoryGenie and get the acceleration in the main code using the get acceleration function?